core-blocks/test/fixtures/core__pullquote.serialized.html

```diff
@@ -1,4 +1,5 @@
 <!-- wp:pullquote -->
 <blockquote class="wp-block-pullquote alignnone">
-	<p>Testing pullquote block...</p><cite>...with a caption</cite></blockquote>
+	<p>Testing pullquote block...</p><cite>...with a caption</cite>
```

Comment:
Unrelated. Ugh, maybe we should wrap cite elements in a footer or something?

Reply:
>maybe we should wrap cite elements in a footer or something?

Would the benefit there be merely to put `cite` on its own line?

Reply:
To me it doesn't make sense to have phrasing content next to block level content (`cite` and `p` on the same level). I guess that's just a personal thing.